lookup_plugins/parse_mt_report.py

```
@@ -0,0 +1,626 @@
+
+def write_opt(content, output_file, include_titles, return_string=False):
+    # Detecting whether it's MT5 report. We need to check if html_content string contains "<?xml" text.
+    is_mt5 = "<?xml" in content
+
+    if is_mt5:
+        return write_opt_mt5(content, output_file, include_titles, return_string=return_string)
+    else:
+        return write_opt_mt4(content, output_file, include_titles, return_string=return_string)
+
+def write_opt_mt5(content, output_file, include_titles, return_string=False):
+    tree = ET.ElementTree(ET.fromstring(content))
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)